Jenkinsfile

```diff
@@ -11,5 +11,34 @@ pipeline {
         sh 'xcodebuild -workspace GiniSwitchSDK/Example/GiniSwitchSDK.xcworkspace -scheme "GiniSwitchSDK-Example" -destination \'platform=iOS Simulator,name=iPhone 6\' test'
       }
     }
+    stage('Hockey deployment') {
+      when {
+        branch 'develop'
```

Comment:
every time when develop builds it gets pushed to hockey, is that intended?

Reply:
I'm not sure. I like the idea of always having an up to date version on Hockey, especially since it's only internal. But that being said, I also like what you have - a special branch you push to.

I'll think about it once I get all credentials working on the build machine.